### PR TITLE
INCIDENT -C826

### PR DESCRIPTION
**Pull request by TeshariEnjoer**

### CODE REVIEW

File: tff_modular/modules/void/code/sub_modules/map/void_turf.dm [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		C.apply_damage(15, BRUTE, BODY_ZONE_CHEST)
+		to_chat(C, span_black("DON'T DO IT!"))
+	user.balloon_alert(user, "RCD gone!")
+	convert_to_void(the_rcd)
+
+
+/turf/open/void_turf/proc/convert_to_void(atom/target)
+	if(istype(target, /mob/living/basic/void_creture) || istype(target, /obj/structure/void_puddle))
+		return
+	target.visible_message(span_black("[target.name] WAS CONSUMED BY VOID!"))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/map/void_turf.dm:46]: target.visible_message(span_userdanger("[target.name] was consumed by void!"))

File: tff_modular/modules/void/code/sub_modules/map/void_turf.dm [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	convert_to_void(user)
+
+/turf/open/void_turf/rcd_act(mob/user, obj/item/construction/rcd/the_rcd)
+	if(issilicon(user))
+		convert_to_void(user)
+		return
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		C.apply_damage(15, BRUTE, BODY_ZONE_CHEST)
+		to_chat(C, span_black("DON'T DO IT!"))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/map/void_turf.dm:38]: span_warning("It's surely a bad idea.")

File: tff_modular/modules/void/code/sub_modules/map/void_turf.dm [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+	user.balloon_alert(user, "RCD gone!")
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/map/void_turf.dm:39]: RCD's gone.

File: tff_modular/modules/void/code/sub_modules/map/void_turf.dm [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+
+/turf/open/void_turf/proc/convert_to_void(atom/target)
+	if(istype(target, /mob/living/basic/void_creture) || istype(target, /obj/structure/void_puddle))
+		return
+	target.visible_message(span_black("[target.name] WAS CONSUMED BY VOID!"))
+
+	if(isliving(target) || issilicon(target))
+		var/mob/living/L = target
+		to_chat(L, span_black("THE VOID CONSUMES YOU, MAKING YOU A PART OF IT."))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/map/void_turf.dm:50]: Void consumes you. You're now one.
span_userdanger

File: tff_modular/modules/void/code/sub_modules/map/void_turf.dm [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+	icon_state = true_icon_state
+	playsound(src, pick('tff_modular/modules/void/sounds/drip1.ogg','tff_modular/modules/void/sounds/drip2.ogg','tff_modular/modules/void/sounds/drip3.ogg'), 90)
+	update_appearance(UPDATE_ICON_STATE)
+	if(!instant)
+		addtimer(CALLBACK(src, PROC_REF(release_void)), 180 SECONDS)
+
+/obj/structure/void_puddle/proc/release_void()
+	if(!src)
+		return
+	visible_message(span_black("[name] is.. peace by reality..."))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/map/void_turf.dm:86]: [name] is gone.

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+/mob/living/basic/void_creture/melee_attack(atom/target, list/modifiers)
+	if(!combat_mode)
+		if(istype(target, /obj/machinery/door/airlock))
+			var/obj/machinery/door/airlock/a = target
+			playsound(a, 'sound/machines/airlock_alien_prying.ogg', 100, TRUE)
+
+			if(a.locked)
+				do_sparks(3, TRUE, a)
+				a.visible_message(span_black("[src.name], beggin corrupt [a.name] bolts with void!"))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:167]: a.visible_message(span_userdanger("[src.name], starts melting through [a.name], slowly dissolving it in void!"))

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+				balloon_alert(src, "Beging crawling bolt!")
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:168]: Opening an airlock,

File: tff_modular/modules/void/code/sub_modules/mob/void_wound.dm [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+
+/**
+ * ВНУТРЕННЯ ПУСТОТА - УЗНАЙ ЧТО ТАКОЕ СТРАДАНИЕ.
+ * Прогресирующий ваунд, в пяти стадиях. Что постепенно уничтожает зараженного.
+ */
+
+/datum/wound/inner_void
+	name = "Void Decay"
+	desc = "Inener void consume you."
+	treat_text = "Threatment impossible. Powerfull painkiller recommendet. \n clinical dead cause body determination."
```
Comment (HWSensum) [inline tff_modular/modules/void/code/sub_modules/mob/void_wound.dm:19]: Вернее будет 

`Inener void consumes you.`

и

`Threatment is impossible. The use of a strong painkiller is recommended. \n Clinical death may cause body determination(?)`

Хотя я не уверен, что значит "решимость тела" во время клинической смерти.

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+				if(!do_after(src, 5 SECONDS, a))
+					balloon_alert(src, "Stand still!")
+					return FALSE
+				do_sparks(3, TRUE, a)
+				a.unlock()
+
+			a.visible_message(span_black("[src.name], beggin force open [a.name]"))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:175]: a.visible_message(span_userdanger("[src.name], starts forcibly opening [a.name]!"))

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 10]
```diff
@@ -0,0 +1,233 @@
...
+			balloon_alert(src, "Beging opening!")
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:176]: Opening an airlock.

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+				return FALSE
+
+			a.open()
+			do_sparks(3, TRUE, a)
+			balloon_alert(src, "Succesfull!")
+			return FALSE
+
+	if(istype(target, /turf/closed/wall))
+		var/turf/closed/wall/w = target
+		w.visible_message(span_black("[src.name], Beggin corrupt [w.name] with void!"))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:189]: w.visible_message(span_userdanger("[src.name] starts melting through the [w.name] dissolving it in void!"))

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+		playsound(w, 'sound/machines/airlock_alien_prying.ogg', 100, TRUE)
+
+		balloon_alert(src, "Beging crushing!")
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:192]: Destroying wall.

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+		return FALSE
+
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		var/obj/item/bodypart/chest/C = H.get_bodypart(BODY_ZONE_CHEST)
+		var/datum/wound/inner_void/infected/void_infection
+		if(!(H.all_wounds & void_infection))
+			void_infection = new()
+			void_infection.apply_wound(C)
+			to_chat(H, span_black("VOID CORRUPT YOU..."))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:209]: Void corrupts you.

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+
+	return ..(target, modifiers)
+
+//Наносим урон тем, кто нас ударит... Пустота делает больно!
+/mob/living/basic/void_creture/attack_hand(mob/living/carbon/human/user, list/modifiers)
+	. = ..()
+	to_chat(user, span_black("Void.. corrupt you..."))
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:216]: Void corrupts you.

File: tff_modular/modules/void/code/sub_modules/mob/void_wound.dm (hunk 8 above)
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/mob/void_wound.dm:19]: Threatment impossible. Using of painkillers highly recommended. Patient death is irreversible.

File: tff_modular/modules/void/code/sub_modules/visual/void_steps.dm [hunk 15]
```diff
@@ -0,0 +1,23 @@
+/obj/effect/temp_visual/void_step
+	name = "Void steps"
+	desc = "Whatch your steps.."
```
Comment (bowlofsoup4ik) [inline tff_modular/modules/void/code/sub_modules/visual/void_steps.dm:3]: Watch your step...

**Review by bowlofsoup4ik — CHANGES_REQUESTED** [review]
пожалуйста скачай дуолинго

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm (hunk 9 above)
Comment (HWSensum) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:175]: `[src.name] starts to force open the [a.name]!`

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm (hunk 6 above)
Comment (HWSensum) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:167]: `[src.name] is corrupting the [a.name] bolts with the void!`

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm [hunk 16]
```diff
@@ -0,0 +1,233 @@
...
+	melee_damage_upper = 70
+	melee_damage_lower = 80
+	mob_size = MOB_SIZE_HUGE
+	armour_penetration = 100
+	pixel_x = -8
+	base_pixel_x = -8
+	maptext_height = 48
+	maptext_width = 48
+	mouse_opacity = MOUSE_OPACITY_ICON
+	death_message = "Oh.. siriosly? You kill that.. soo you need just know.. teshari the best.. ye-a... how did you do it?"
```
Comment (HWSensum) [inline tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:33]: А это что?

**Review by HWSensum — CHANGES_REQUESTED** [review]
Я тоже пару косяков с текстом заметил.

File: tff_modular/modules/void/code/sub_modules/mob/void_mob.dm (hunk 16 above)
Comment (TeshariEnjoer) [reply tff_modular/modules/void/code/sub_modules/mob/void_mob.dm:33]: Абсолютно обычное сообщение выводящееся при смерти абсолютно обычного существа.

**Pull request closed without merging**